docs/releaseNotes.yml

```diff
@@ -84,6 +99,20 @@ items:
           <code>*-migration.yaml</code> files have not been part of the migration instructions, and
           while the <code>*-agent.yaml</code> files remained part of the instructions they were
           actually unnescessary.
+      - title: Metric naming change for grpc_stats 
+        type: change
+        body: >-
+          The previous version of $productName$ was based on Envoy 1.17 and when using grpc_stats
+          with <code>all_methods</code> or <code>services</code> set, it would output metrics in
+          the following format <code>envoy_cluster_grpc_{ServiceName}_{statname}</code>. When
+          neither of these fields are set it would be aggregated to <code>envoy_cluster_grpc_{statname}</code>.
+          
+          The new behavior since Envoy 1.18 will produce metrics in the following format
+          <code>envoy_cluster_grpc_{MethodName}_statsname</code> and <code>envoy_cluster_grpc_statsname</code>. 
```

Comment:
That's not quite true.  The new name only ends up using `{BaseMethodName}` if there is exactly 1 dot in the qualified service name.

Reply:
I've spent some time trying to succinctly explain it, but I'm not sure it's possible. I think the call is to link to a docs page that can take some more space to explain it.

Reply:
yeah lets update it on docs/release